src/components/Session/SessionFilter.vue

```diff
@@ -105,6 +111,36 @@ export default defineComponent({
       router.back()
     }
 
+    const checkMySchedule = () => {
+      const events: any[] = [];
+      favoriteSessions.value.forEach(function(SessionId) {
+        const session = computed(() => getSessionById(SessionId));
+        events.push({"startTime": new Date(session.value.start), "endTime": new Date(session.value.end)});
+      });
+      
+      var isConflict = false;
```

Comment:
```suggestion
      let isConflict = false;
```

用 [let](https://developer.mozilla.org/zh-TW/docs/Web/JavaScript/Reference/Statements/let) 來宣告區域變數。